addons/hide-flyout/userscript.js

```diff
@@ -0,0 +1,26 @@
+export default async function ({ addon, global, console }) {
+  while (true) {
+    let flyOut = await addon.tab.waitForElement(".blocklyFlyout", { markAsSeen: true });
+    let state = true;
+    function mouseMove(e) {
+      let sidePos = document.querySelector(".blocklyToolboxDiv").getBoundingClientRect().x;
+      let scrollBar = document.querySelector(".blocklyFlyoutScrollbar");
+      if (e.clientX - sidePos < 310) {
```

Comment:
What's 310? This feels like it will change screen to screen.

Reply:
I think that it should be the same everywhere... This is the width of the flyout.